lib/widgets/my_alert_dialog.dart

```diff
@@ -0,0 +1,91 @@
+import "package:flutter/material.dart";
+
+import "../../../config/ui_config.dart";
+import "../../../theme/app_theme.dart";
+import "../../../utils/context_extensions.dart";
+
+Future<void> showCustomDialog({
+  required BuildContext context,
+  required VoidCallback onConfirmTapped,
+  required String confirmText,
+  required Widget dialogContent,
+}) async {
+  await showDialog<void>(
+    context: context,
+    builder: (BuildContext context) {
+      return LayoutBuilder(
+        builder: (context, constraints) {
+          final double maxWidth = MediaQuery.of(context).size.width * 0.89;
+          return Center(
+            child: ConstrainedBox(
+              constraints: BoxConstraints(
+                maxWidth: maxWidth,
+              ),
+              child: _MyAlertDialog(
+                dialogContent: dialogContent,
+                onConfirmTapped: onConfirmTapped,
+                confirmText: confirmText,
+              ),
+            ),
+          );
+        },
+      );
+    },
+  );
+}
+
+class _MyAlertDialog extends StatelessWidget {
+  final Widget dialogContent;
+  final VoidCallback onConfirmTapped;
+  final String confirmText;
+
+  const _MyAlertDialog({
+    required this.dialogContent,
+    required this.onConfirmTapped,
+    required this.confirmText,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      content: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: AboutUsConfig.dialogHorizontalPadding,
```

Comment:
Since dialog's now global,  wdyt about moving that to separate config file?

Reply:
Yes, you're right! So I'll move it to new config class like `AlertDialogConfig`